papers/ivan_perez_avellaneda/myst.yml

```
@@ -0,0 +1,54 @@
+version: 1
+extends: ../papers.yml
+project:
+  # Update this to match `scipy-2025-<folder>` the folder should be `<firstname_surname>`
+  id: scipy-2025-myst_template
+  # Ensure your title is the same as in your `main.md`
+  #title: 'Paper: SciPy Optimize and Reachability of Nonlinear Systems'
+  title:'CFSpy: A Python Library for the Computation of Chen-Fliess Series'
+  # subtitle:
+  description: Add a two sentence description about your article.
+  # Authors should have affiliations, emails and ORCIDs if available
+  authors:
+    - name: Ivan Perez Avellaneda
+      email: iperezave@gmail.com
+      orcid: 0000-0003-3127-8454
+      affiliations:
+        - Monaghan Medical Corp.
+#        - Egyptian Embassy, S.P.Q.R.
+#    - name: Mark Anthony
+#      email: mark37@rome.it
+#      affiliations:
+#        - Egyptian Embassy, S.P.Q.R.
+#    - name: Jarrod Millman
+#      email: millman@rome.it
+#      affiliations:
+#        - Egyptian Embassy, S.P.Q.R.
+#        - Yet another place, S.P.Q.R.
+#      corresponding: true
+#    - name: Cleopatra
+#      email: cleopatra@rome.it
+#      affiliation: Ptolemaic Kingdom
+  keywords:
+    - Chen-Fliess series
+    - Nonlinear Control Systems
+    - Python
+  # Add the abbreviations that you use in your paper here
+  abbreviations:
+    MyST: Markedly Structured Text
+  # It is possible to explicitly ignore the `doi-exists` check for certain citation keys
+  error_rules:
+    - rule: doi-exists
+      severity: ignore
+      keys:
+        - Atr03
+        - terradesert
+        - jupyter
+        - sklearn1
+        - sklearn2
```

Automated checks are failing as these three citations do not have DOIs listed. If the references do not actually have DOIs at all, you may add them to the list to allow the checks to pass!

```suggestion
        - Mitchell2007
        - Bullo2022
        - Perez2024
```

---

Thanks, that worked